eva-events/src/main/kotlin/com/razz/eva/events/SharedMutableFlowEventBus.kt

```diff
@@ -0,0 +1,64 @@
+package com.razz.eva.events
```

Comment:
I suggest to move this class to `uow` module. Current `eva-events` module is super simple (event class + interfaces) and if someone wants to develop his own consumer/publisher implementation he doesn't need this class in his classpath

Reply:
Agree with arguments regarding minimalism in `eva-events` however `uow` doesn't looks like a proper place to me.